WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewController.swift

```diff
@@ -81,10 +121,9 @@ private extension CustomerSelectorViewController {
             "Add customer details",
             comment: "Title of the order customer selection screen."
         )
-    }
 
-    enum Constants {
-        static let pageSize = 25
-        static let firstPageNumber = 1
+        static let genericAddCustomerError = NSLocalizedString("Failed to fetch the customer data. Please try again.",
+                                                                comment: "Error message in the Add Customer to order screen " +
+                                                                "when getting the customer information")
```

Comment:
To avoid extra spaces and the 160 lint warning, this works too: 
```suggestion
        static let genericAddCustomerError = NSLocalizedString(
            "Failed to fetch the customer data. Please try again.",
            comment: "Error message in the Add Customer to order screen when getting the customer information")
```